test/system/helpers.systemd.bash

```diff
@@ -14,12 +14,44 @@ fi
 
 mkdir -p $UNIT_DIR
 
+journalctl_support_user() {
+    r_value=0
+    run systemd-analyze cat-config systemd/journald.conf | grep Storage
```

Comment:
The pipe does not capture output here at all. $output is set by the run function and that does not output to stdout by default so the pipe cannot work.